webhook.yml

```
@@ -370,6 +384,10 @@ components:
               description: "Length of video file (milliseconds)"
             contentProvider:
               $ref: "#/components/schemas/ContentProvider"
+            quoteToken:
+              type: string
+              description: |+
+                Quote token of the message. Only included when a message object that can be specified as a quote target was sent as a push message.
```

```suggestion
                Quote token of the message. Only included when a message object that can be specified as a quote target was sent.
```
(For all`quoteToken.description` in `webhook.yml`)